pipeline/outputs/loki.md

```diff
@@ -176,6 +177,26 @@ Based in the JSON example provided above, the internal stream labels will be:
 job="fluentbit", team="Santiago Wanderers"
 ```
 
+### Structured Metadata
+
+[Structured metadata](https://grafana.com/docs/loki/latest/get-started/labels/structured-metadata/) allows attaching custom fields to individual log lines without needing to embed the information in the content of the log line. This is particularly useful for high cardinality data for which labels are ill-suited. Although technically not labels, from a configuration perspective the `structured_metadata` configuration parameter behaves similarly to the `labels` parameter: both are comma-delimited `key=value` lists, and both may use record accessors to reference keys within the record being processed.
+
+Consider the following example:
+
+```python
+[OUTPUT]
+    name                loki
+    match               *
+    labels              cluster=my-k8s-cluster, region=us-east-1, namespace=$kubernetes['namespace_name']
+    structured_metadata pod=$kubernetes['pod_name']
+```
+
+This configuration defines fixed values for the cluster and region labels, uses the record accessor pattern to set the namespace label to the namespace name as determined by the kubernetes metadata filter (not shown in the example), and uses a structured metadata field to hold the pod name.
+
+Other common uses for structured metadata include trace and span ids, process and thread ids, and log levels.
+
+Structured metadata is only officially supported starting with Loki 3.0 and should not be used with Loki deployments that lack support.
```

Comment:
```suggestion
Structured metadata is supported starting with Loki 3.0, and shouldn't be used with
Loki deployments prior to Loki 3.0.
```

Reply:
@lockewritesdocs The reason for my choice of language was that structured metadata *did* work in Loki prior to v3.0 (since 2.9.0) but it didn't go GA until then.  Many Loki deployments used structured metadata prior it being GA -- I even had a Grafana engineer tell me they've been using it Grafana Cloud for a good while and even though the docs hadn't been updated yet it was highly unlikely to change.  But the feature was there, and the config option to enable it was also there, it was just marked as experimental.

In light of that, do you think it's appropriate to reword, or continue with what you proposed?

Reply:
@jtackaberry, appreciate the explanation! Given that the feature was officially supported in Loki 3.0, I think we can say:

>Structured metadata is officially supported starting with Loki 3.0, and shouldn't be used with Loki deployments prior to Loki 3.0.

That statement says, "You shouldn't use this feature with Loki deployments prior to 3.0," but that doesn't mean you _can't_ use it. Just more of a "Use at your own risk" scenario. I updated my suggested change to include "officially supported" -- does that work for you?

Reply:
Yep, cheers!